modules/core/04-channel/types/upgrade.go

```diff
@@ -35,20 +35,6 @@ func NewUpgradeTimeout(height clienttypes.Height, timestamp uint64) UpgradeTimeo
 	}
 }
 
-// ValidateBasic performs a basic validation of the upgrade fields
-func (u Upgrade) ValidateBasic() error {
```

Comment:
follow up to [this discussion](https://github.com/cosmos/ibc-go/pull/3451#discussion_r1168870480)

Reply:
>  I guess we could keep if for the situation where the entire counterparty upgrade type is provided in the TRY and ACK, in this situation the sequence should not equal 0.

Was chatting with @colin-axner about this and it may still be useful to keep this around for validating the counterparty upgrade.